AU2/plugins/custom_plugins/WantedPlugin.py

```diff
@@ -118,6 +118,22 @@ def on_event_update(self, e: Event, htmlResponse) -> List[HTMLComponent]:
         e.pluginState[self.identifier] = htmlResponse[self.event_html_ids["Wanted"]]
         return [Label("[WANTED] Success!")]
 
+    def render_event_summary(self, event: Event) -> List[AttributePairTableRow]:
+        results = []
+        for playerID in event.pluginState[self.identifier]:

```

Comment:
I've just discovered that this throws an error for the gigabolt events....

```suggestion
        for playerID in event.pluginState.get(self.identifier, ()):
```